src/RoslynAnalyzers/Microsoft.CodeAnalysis.BannedApiAnalyzers/RulesMissingDocumentation.md

```diff
@@ -2,6 +2,4 @@
 
 Rule ID | Missing Help Link | Title |
 --------|-------------------|-------|
-RS0030 | <https://github.com/dotnet/roslyn/blob/main/src/RoslynAnalyzers/Microsoft.CodeAnalysis.BannedApiAnalyzers/BannedApiAnalyzers.Help.md> | Do not use banned APIs |
```

Comment:
I assume I should revert these files? I didn't manually change them

Reply:
@JoeRobich do you know? IIRC you had this in your PR too

Reply:
These files are automatically generated during the build. You are fine to merge them. Sorry it is noisy.